config/samples/osbuilder_v1alpha1_osbuildconfig.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: osbuilder.project-flotta.io/v1alpha1
+kind: OSBuildConfig
+metadata:
+  name: osbuildconfig-sample
+spec:
+  details:
+    distribution: Fedora
```

Comment:
Distribution is lower case and includes the version. E.g. `fedora-35`

Reply:
should there be a validation for that field structure in the CRD?
what would be the desired pattern for it? we can add on top of https://github.com/project-flotta/osbuild-operator/blob/main/api/v1alpha1/osbuildconfig_types.go#L41 something like:
```
// +kubebuilder:validation:Pattern=^[a-z0-9-.]+$
```

is this field validate on the composer API side and has a known format to it?

Reply:
There is no validation for it on the Composer side either, nor is there in the Weldr API (or defined in the Blueprint document https://www.osbuild.org/guides/blueprint-reference/blueprint-reference.html#distribution-selection-with-blueprints)